tests/sentry/grouping/snapshots/test_grouphash_metadata/test_metadata_from_variants/newstyle@2023_01_11/unreal_event_capture_mac.pysnap

```diff
@@ -24,96 +24,181 @@ metrics with tags: {
 ---
 contributing variants:
   app*
-    hash: "9e04decaf79ecba9dc0314dc0edd3993"
+    hash: "54e8028fb2526cf31b12dd66c01ad9e2"
```

Comment:
We now have system and app hashes:
<img width="362" alt="image" src="https://github.com/user-attachments/assets/b6955f54-2432-4a9d-80e9-a0554875a7c2" />